samples/bluetooth/mesh/light/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
+set(PM_STATIC_YML_FILE
```

Comment:
if exists

Reply:
maybe

Reply:
I took it from `peripheral_lbs` sample. I was also wondering if we should add if exists there. If it doesn't exist, nothing bad will happen, so i accepted it there.

Reply:
In `partition_manager.cmake`, there are checks, so the file is used only if it exists and it's not a directory.
I think we could use it in that way for all of the samples.